auditlog/models.py

```diff
@@ -6,15 +6,18 @@
 from django.conf import settings
 from django.contrib.contenttypes.fields import GenericRelation
 from django.contrib.contenttypes.models import ContentType
+from django.core import serializers
 from django.core.exceptions import FieldDoesNotExist
 from django.db import DEFAULT_DB_ALIAS, models
 from django.db.models import Q, QuerySet
 from django.utils import formats, timezone
 from django.utils.encoding import smart_str
 from django.utils.translation import gettext_lazy as _
 
+from .history import HistoricalStateLookupManagerMixin
 
-class LogEntryManager(models.Manager):
+
+class LogEntryManager(HistoricalStateLookupManagerMixin, models.Manager):
```

Comment:
The idea behind putting a mixin here is so that if you don't want to include the historical state lookups in the manager, It will be easier for me to maintain a separate fork with those features included if the distinct code is in a different file.